torch/csrc/distributed/c10d/nvshmem_extension.cu

```diff
@@ -452,4 +450,24 @@ at::Tensor nvshmem_sendrecv(
   return out;
 }
 
+at::Tensor nvshmem_all_to_all(
+    at::Tensor& input,
+    at::Tensor& out,
+    std::string group_name) {
```

Comment:
Does group_name need to be a string by value or can it be a const ref or stringview?

Reply:
Yeah I tried, but dispatcher signature registration does not seem to support `const std::string&`